### PR TITLE
Permissions for wf actions

### PR DESCRIPTION
**Pull request by ghost**
added pod and wftmpl permissions

### CODE REVIEW

**Review by oren-codefresh — APPROVED** [review]

File: manifests/app-proxy/app-proxy.crb.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: cap-app-proxy
+  kind: ClusterRole
+  name: argo-server-cluster-role
 subjects:
   - kind: ServiceAccount
     name: cap-app-proxy
+    namespace: default
```
Comment (roi-codefresh) [inline manifests/app-proxy/app-proxy.crb.yaml:14]: how will this work?

Comment (ghost) [reply manifests/app-proxy/app-proxy.crb.yaml:14]: I tested it locally - I coped this file to the local clone of my runtime repo, and ran `kustomize build` - the resulting manifests contained this CRD with the correct namespace of my runtime (the one defined in the `kustomization.yaml` file).

Comment (roi-codefresh) [reply manifests/app-proxy/app-proxy.crb.yaml:14]: oh, right, kustomize replaces `default` with the compiled namespace

**Review by roi-codefresh — APPROVED** [review]

**Pull request merged**